docs/user/get-started.md

```diff
@@ -46,8 +48,8 @@ brew install hdf5
 
 ### Deploy
 
-This section shows how to deploy Vald with Scylla using Helm.
-Scylla is used as a datastore for index-metadata and backup-manager.
+This chapter will show you how to deploy using Helm and run Vald on your k8s cluster.<br>
+This chapter uses Scylla DB as the backend data store for index-metadata and backup-manager.<br>
```

Comment:
Start thinking it is a get started page and user may not know what is backup manager. we should use understandable work here :)
```suggestion
This chapter uses Scylla DB as a backend data store for indexing and data backup.<br>
```